graylog2-server/src/main/java/org/graylog2/inputs/codecs/GelfCodec.java

```diff
@@ -179,6 +179,11 @@ public Message decode(@Nonnull final RawMessage rawMessage) {
                 continue;
             }
 
+            // Do not index useless GELF "version" field.
+            if ("version".equals(key)) {
+                continue;
+            }
+
```

Comment:
This needs to be done before removing the `_` character from additional fields. ([here](https://github.com/Graylog2/graylog2-server/blob/do-not-index-gelf-version-field/graylog2-server/src/main/java/org/graylog2/inputs/codecs/GelfCodec.java#L171))

Otherwise a `version` cannot be set at all.